app/services/performance.ts

```diff
@@ -50,15 +52,34 @@ export class PerformanceService extends StatefulService<IPerformanceState> {
           this.droppedFramesDetected.next(stats.percentageDroppedFrames / 100);
         }
 
-        stats.CPU += am
+        const electronCpu = am
           .map(proc => {
             return proc.cpu.percentCPUUsage;
           })
           .reduce((sum, usage) => sum + usage);
 
+        if (this.measurements != null) {
+          this.measurements.push(electronCpu);
+        }
+
+        if (this.measurements && this.measurements.length >= 60) {
+          console.log(
+            `Average Electron CPU over ${this.measurements.length} measurements:`,
+            this.measurements.reduce((sum, meas) => sum + meas, 0) / this.measurements.length,
+          );
+
+          this.measurements = null;
+        }
+
+        stats.CPU += electronCpu;
+
         this.SET_PERFORMANCE_STATS(stats);
       },
     );
+
+    window['startMeasure'] = () => {
+      this.measurements = [];
+    };
```

Comment:
I'll be backing these changes out of this PR.  Was just using this to sanity check the performance increases I was seeing.